Rakefile

```diff
@@ -4,7 +4,11 @@ rescue LoadError
   puts "You must `gem install bundler` and `bundle install` to run rake tasks"
 end
 
-APP_RAKEFILE = File.expand_path("../test/dummy/Rakefile", __FILE__)
+gemfile = ENV["BUNDLE_GEMFILE"] # i.e /home/blahblah/apps/skroutz/cogy/gemfiles/4.2.gemfile
+rails_major_version = gemfile[gemfile.rindex('/') + 1 ... gemfile.rindex('.')]
```

Comment:
Also do this in one line (get rid of the `gemfile` variable):

```ruby
rails_version = File.basename(ENV["BUNDLE_GEMFILE"], ".gemfile")
```

P.S. The minor version is also included in the value.